README.md

```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
 _To get help, [submit an issue](https://github.com/jrnl-org/jrnl/issues/new) on
 Github._
 
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
-folder for instant syncing and you can be assured that your journal will still
-be readable in 2050, when all your fancy iPad journal applications will long be
-forgotten.
+`jrnl` is a simple journal application for the command line.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+Its goal is to facilitate the rapid creation and viewing of journal entries. It
+is flexible enough to support different use cases and organization strategies.
+It is powerful enough to search through thousands of entries and display, or
+"filter," only the entries you want to see.
 
-### Why keep a journal?
+`jrnl` includes support for [128-bit AES
+encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard) using
+[cryptography.Fernet](https://cryptography.io/en/latest/fernet/).
 
-Journals aren't just for people who have too much time on their summer
-vacation. A journal helps you to keep track of the things you get done and how
-you did them. Your imagination may be limitless, but your memory isn't. For
-personal use, make it a good habit to write at least 20 words a day. Just to
-reflect what made this day special, or why you haven't wasted it. For
-professional use, consider a text-based journal to be the perfect complement to
-your GTD todo list - a documentation of what and how you've done it.
+## In a Nutshell
 
-In a Nutshell
--------------
+To make a new entry, just enter
 
-To make a new entry, just type
+``` sh
```

Comment:
Using `sh` here adds syntax highlighting that we don't really want (`in` and `time` are now showing up in a different color.